src/wheatley.ts

```diff
@@ -948,6 +956,9 @@ export class Wheatley {
     // command edit/deletion
 
     register_text_command(trigger: Discord.Message, command: TextBasedCommand, deletable = true) {
+        if (this.passive) {
```

Comment:
I am not sure this is the right place to do this. Maybe instead this could be checked in `handle_text_command`/`handle_slash_comand`? It may also be important to do this in `on_interaction`, depending on goals.

Reply:
If we do this not here but later on, wouldn't that mean that the commands are still registered though?

Reply:
OK, I missed that you meant to only do this for text commands. I'll update the PR.

Reply:
Talked on discord, thanks for resolving